lib/cli.js

```diff
@@ -2,6 +2,14 @@ import yargs from 'yargs'
 import {log} from './utils/log'
 import {version} from '../package.json'
 
+// Workaround to remove display of script name in help. See:
+// * https://github.com/yargs/yargs/pull/1143
+// * https://github.com/yargs/yargs/issues/1065
+// * https://github.com/yargs/yargs/issues/1048
+function fixHelp (output) {
+  return output.replace(/( +)contentful\.js/g, '$1')
+}
+
```

Comment:
Nasty ugly bug adding `contentful.js` to our help texts. This hack removes that for now, should be fixed soonish by the community ™️